draft-ietf-wimse-s2s-protocol.md

```diff
@@ -233,14 +234,15 @@ The decoded JWT claims of the WIT from the example above are shown here:
 {
   "cnf": {
     "jwk": {
+      "alg": "EdDSA",
```

Comment:
Note for myself from SPIFFE sig-spec: This restricts WIT to be used with JOSE only.

Reply:
Per the pointer that @jricher gave in https://github.com/ietf-wg-wimse/draft-ietf-wimse-s2s-protocol/pull/115/files#r2070242131 to  [RFC9421-3.3.7](https://www.rfc-editor.org/rfc/rfc9421.html#name-json-web-signature-jws-algo) - I don't think it actually does restrict the WIT to be used with JOSE only. No?

Reply:
It would effectively restrict it to use JOSE-defined algs but the HTTPSig RFC at least says how to map a JOSE JWS alg to the HTTPSig primitives (`HTTP_SIGN` and `HTTP_VERIFY`) so for that case you're OK. It doesn't necessarily allow for future usages with other presentation mechanisms but I think that's fine.